plugins/pulp_docker/plugins/importers/importer.py

```diff
@@ -76,7 +76,6 @@ def sync_repo(self, repo, sync_conduit, config):
         :return: report of the details of the sync
         :rtype:  pulp.plugins.model.SyncReport
         """
-        repo = model.Repository.objects.get_repo_or_missing_resource(repo.id)
```

Comment:
If we want to use the new-style object everywhere, what is the advantage of passing the old-style object in?